peers/info_client.go

```diff
@@ -47,7 +48,7 @@ func (i *InfoAPI) GetNodeID(ctx context.Context) (ids.NodeID, *signer.ProofOfPos
 	return i.client.GetNodeID(ctx, i.options...)
 }
 
-func (i *InfoAPI) GetNodeIP(ctx context.Context) (string, error) {
+func (i *InfoAPI) GetNodeIP(ctx context.Context) (netip.AddrPort, error) {
```

Comment:
Are we able to change the name of this function?

Reply:
It's not part of an interface, but our `InfoAPI` type is written as an extension to `InfoClient` that embeds the rpc options in the object, rather than leaving it to the caller. As such, matching the `InfoClient` interface makes the code more readable.

What's the motivation for changing the function name?

Reply:
The function is named `GetNodeIP` but we're returning both the IP and the port

Reply:
My preference would be to keep parity with the underlying client interface, even if the name could be improved. That greatly helps code discovery when referencing existing usages as examples.